libnetwork/cni/config_test.go

```diff
@@ -633,7 +633,7 @@ var _ = Describe("Config", func() {
 			// reload configs from disk
 			libpodNet, err = getNetworkInterface(cniConfDir)
 			Expect(err).To(BeNil())
-			// check the the networks are identical
+			// check the etworks are identical
```

Comment:
misspelled "etworks"

Reply:
That's quite funny as this PR should enable a missspell linter :angel: